operator/src/main/resources/scripts/introspectDomain.sh

```diff
@@ -56,6 +56,11 @@ source ${SCRIPTPATH}/utils.sh
 
 traceTiming "INTROSPECTOR '${DOMAIN_UID}' MAIN START"
 
+# 
+# if the common mount feature is active, verify the mount, and log mount information
+#
+checkCommonMount || exit 1
+
```

Comment:
Minor: Consider adding "checkEnv WDT_MODEL_HOME WDT_INSTALL_HOME || exit 1" at line 166, inside the 'FromModel' if clause.  This will assert that the env vars are set and implicitly create FINE trace statements for each.